scripts/update/Updater.php

```diff
@@ -73,5 +73,10 @@ public function update($currentVersion)
 
             $this->setVersion('1.0.0');
         }
+
+        if($this->isVersion('1.0.0')){
+            call_user_func(new RegisterPci(), []);
```

Comment:
would be better to merge this after https://github.com/oat-sa/extension-tao-itemqti-pci/pull/44 to benefit from individualised pci registration script